### ISSUE TITLE
Variant information panel

### ISSUE THREAD

**Issue by AlistairNWard** [iobio/gene.iobio.vue#200]
This panel can get super full now and hard to parse. Would it be worth condensing this to a few important pieces of information, and allow a modal to open that includes everything? The modal would be far less constrained for space, so we could present all this info in a much clearer way.

**Comment by tonydisera** [comment]
Agreed.  I was also wondering about making the gene and variant detail expand (and retract) when a variant is selected.  I'll put both approaches out there for you to play with.

**Comment by tonydisera** [comment]
https://docs.google.com/presentation/d/103p4eeCFoeyVzxpVKkvC5-l2uQXKITFVssd5jD0swUk/edit?usp=sharing

New Variant Detail Card
1. Impact for Non-canonical transcript
2. Quality
3. Pedigree
4. More... for Allele Freq

**Comment by mvelinder** [comment]
Done
